bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/AbstractTestCreateTable.java

```diff
@@ -135,6 +130,27 @@ private boolean contains(TableName[] tableNames, TableName tableNameTocheck) {
     return false;
   }
 
+  private void assertBadTableName(String tableName){
+    boolean failed = false;
+    try {
+      createTable(TableName.valueOf(tableName));
+    } catch (Exception ex) {
```

Comment:
Add this in the try block:

```
 Assert.fail("Should fail as table name: '" + tableName + "'");
```

and remove the `assertTrue`

Reply:
Addressed feedback. Please have an fresh look.